From e17895b9298a2cc8dc90a5e6ba2283913a842ff5 Mon Sep 17 00:00:00 2001
From: Vladimir <VladimirJ45>
Date: Sun, 9 Jan 2022 21:08:57 +0200
Subject: [PATCH] updated first file

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 0b669b6..8a44d37 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-1
-2
+update
+update
 3
 4
 5
-- 
2.33.0.windows.2

